### PR TITLE
Adding issue template

### PR DESCRIPTION
**Pull request by jwallra**

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,18 +1,31 @@
...
+1.
+2.
+
+### **Details**
+
+1. Which SDK version are you using?
+    - e.g. 1.5.0
+2. Which OS version did you experience the issue on?
+    - e.g. Android 8.1.0, iOS 11
+3. What device version did you see this error on?  Were you using an emulator or a physical device?
```
Comment (dhei) [inline ISSUE_TEMPLATE.md]: nitpick - change "emulator" to "emulator / simulator"

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,18 +1,31 @@
...
+    - e.g. Google Pixel 2 emulator, iPhone X physical device
+4. What third party libraries are you using?
+    - example
+5. Run the following command and paste the output below: `react-native info`
+
+Place output here
+
+6. Run the following command and paste the output below: `pod --version`
```
Comment (dhei) [inline ISSUE_TEMPLATE.md]: "If you develop for react native iOS, run the following command and paste the output below: `pod --version`"

File: ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -1,18 +1,31 @@
...
+
+Place output here
+
+7. Please enable verbose logging for your app using `AppCenter.setLogLevel(Log.VERBOSE)` before your call to `AppCenter.start(...)` and include the logs here:
```
Comment (dhei) [inline ISSUE_TEMPLATE.md]: This instruction is not the right for react native, please rephrase the instruction from https://docs.microsoft.com/en-us/appcenter/sdk/other-apis/react-native#adjust-the-log-level:

> In iOS, call [MSAppCenter setLogLevel: MSLogLevelVerbose]; before any call to [AppCenterReactNative register]; (or AppCenterReactNativeAnalytics or AppCenterReactNativeCrashes or AppCenterReactNativePush) in AppDelegate.m. You have to add @import AppCenter; if missing in that file. 
> In Android, call AppCenter.setLogLevel(android.util.Log.VERBOSE); before SoLoader.init in onCreate callback in MainApplication.java.

File: ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -1,18 +1,31 @@
-<!--
-  Thanks for your interest in using the App Center SDK for React Native.
+### **Description**
 
-  If you have an issue in React Native platform but have not integrated
```
Comment (dhei) [inline ISSUE_TEMPLATE.md]: @elamalani IMHO I think we should still keep this clarification about using intercom for support tickets.

File: ISSUE_TEMPLATE.md (hunk 1 above)
Comment (jwallra) [reply ISSUE_TEMPLATE.md]: I'll leave it to match the other ones.  The meaning should be clear.

**Pull request merged**